site/404.html

```diff
@@ -8,10 +8,59 @@ <h1>404</h1>
 
 <div class="container">
 	<h2>Sorry! That page doesn't exist!</h2>
+	<p id='closest'></p>
 	<p>Here are some things you can do:</p>
 	<ul>
 		<li><a href="https://github.com/ScratchAddons/website/issues">Create a new issue suggesting it</a></li>
 		<li><a href="/">Return to the home page</a></li>
 		<li><a href="https://findtheinvisiblecow.com/">Find the invisible cow</a></li>
 	</ul>
 </div>
+<script>
+	// Function for load
+	
+	//Strip '.html' and leading '/'s from the pathname
+	var url = window.location.pathname;
+	url = url.replace('.html', '');
+	url = url.replace(/\//, '');
+  if(editDistance(closest(url), url) < 3) {
+    window.location.href = `https://scratchaddons.com/${closest(url)}`;
+  }
+	var newurl = closest(url);
+	document.getElementById(
+		'closest'
+	).innerHTML = `Did you mean "<a href='${newurl}.html'>${newurl}.html</a>"?`;
+	if (url === '404') {
+		document.getElementById('closest').innerHTML =
+			'Why would you purposely navigate to this page.';
+	}
+	// Functions
+	function closest(str) {
+		var urls = ['index', 'blog', 'projects'];
```

Comment:
lol I think you're supposed to change this

Reply:
Yeah, obviously XD. This is the array of correct pages.

Reply:
> You should parse the sitemap to get the links

Wait, where do I find that?